sdk/eventhub/azure-eventhub/README.md

```diff
@@ -40,16 +40,6 @@ There, you can also find detailed instructions for using the Azure CLI, Azure Po
 
 Interaction with Event Hubs starts with an instance of the EventHubClient class. You need the host name, SAS/AAD credential and event hub name to instantiate the client object.
 
-#### Obtain a connection string
-
-For the Event Hubs client library to interact with an Event Hub, it will need to understand how to connect and authorize with it.
-The easiest means for doing so is to use a connection string, which is created automatically when creating an Event Hubs namespace.
-If you aren't familiar with shared access policies in Azure, you may wish to follow the step-by-step guide to [get an Event Hubs connection string](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string).
-
-#### Create client
-
-There are several ways to instantiate the EventHubClient object and the following code snippets demonstrate two ways:
-
 **Create client from connection string:**
 
 ```python
```

Comment:
Add a code snippet for EventHubProducerClient?
The consumer client needs consumer group, but the producer client doesn't need.

Reply:
producer added